internal/versions/tests_utils.go

```diff
@@ -0,0 +1,14 @@
+package versions
+
+import "strings"
+
+// SplitStringToInterfacesArray returns an array of interfaces from a command string
+func SplitStringToInterfacesArray(str string) []interface{} {
```

Comment:
where is it used?

Reply:
This function already exist on https://github.com/openshift/assisted-service/blob/dfec6b83424beea48bac5daaa4cb7c963c619479/internal/oc/release_test.go#L281-L289
maybe it should be moved under `common` or `pkg`